spec/like_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do

```

Comment:
- [ ] All methods in your Like model should be tested as mentioned in the project requirements. Please write a test for the method `update_counter_likes`